### PR TITLE
sync only on pause

### PR DESCRIPTION
**Pull request by mhashemm**

### CODE REVIEW

File: mpv/mpv.go [hunk 1]
```diff
@@ -44,20 +51,31 @@ type Request struct {
...
+		playbackTime, err := strconv.ParseFloat(event.Data, 64)
+		if err != nil {
+			return err
+		}
+		playbackTime = math.Floor(playbackTime)
+		if s.playbackTime == playbackTime {
+			return errNoChange
+		}
 	}
+	return nil
```
Comment (shehabadel) [inline mpv/mpv.go:78]: Can you elaborate what are you doing here?

Comment (mhashemm) [reply mpv/mpv.go:78]: yes i can

Comment (shehabadel) [reply mpv/mpv.go:78]: thank you i am waiting

Comment (mhashemm) [reply mpv/mpv.go:78]: we get the current playback time from the player, and we compare it to the latest playback if it's within the same second we skip this event. this to keep track of the current second we're playing.
actually setting `s.playbackTime = playbackTime` is missing

**Pull request merged**